src/modules/attachMethods.ts

```diff
@@ -0,0 +1,35 @@
+import { Attach } from '@prisma/client';
+import { TRPCError } from '@trpc/server';
+
+import { prisma } from '../utils/prisma';
+
+import { CreateAttach } from './attachTypes';
+import { tr } from './modules.i18n';
+
+const create = async (data: CreateAttach): Promise<Attach> => {
+    return prisma.attach.create({ data });
+};
+
+const getById = async (id: string) => {
+    const attach = await prisma.attach.findFirst({
+        where: { id },
+    });
+
+    if (attach === null) {
+        throw new TRPCError({ code: 'NOT_FOUND', message: tr('Attach not found') });
+    }
+
+    return attach;
+};
+
+const deleteAttach = async (id: string): Promise<Attach> => {
+    await getById(id);
+
+    return prisma.attach.update({ where: { id }, data: { deletedAt: new Date() } });
+};
+
+export const attachMethods = {
+    create,
```

Comment:
lets move body of functions inside the object itself, this way it's one less click when you got from usage to definition in IDE